www/js/lib/uiUtil.js

```diff
@@ -428,6 +431,13 @@ define(rqDef, function() {
         var contentTheme = theme.replace(/^[^_]*/, '');
         var oldAppTheme = oldTheme.replace(/_.*$/, '');
         var oldContentTheme = oldTheme.replace(/^[^_]*/, '');
+        // Hide any previously displayed help
+        var oldHelp = document.getElementById(oldContentTheme + '-help');
+        if (oldHelp) oldHelp.style.display = 'none';
+        // Show any specific help for selected contentTheme
+        var help = document.getElementById(contentTheme + '-help');
+        if (help) help.style.display = 'block';
+        if(theme === 'light' + selectedThemeContent) contentTheme = null;
```

Comment:
This line looks like a fudge. First of all, above, you set `theme` to an invalid value (e.g. `light_mwInvert`), and here you remove the invalid part. It would be much better, I believe, if you just to set to a valid values above (e.g., just set to `light`). Also, note inconsistent spacing after `if` here.

Reply:
So this is why it's best not to set invalid values in the first place! I think this needs to be tweaked so you set the correct values above, then you won't need to correct them, and you won't need to move this code.

Reply:
But then I'm unable to find any other way to get the help wikimedia to DOM when the auto(wikimedia) changes to light because it's easier to display that help on the screen but the removal is difficult if there's another way. So that's why I've done this. And how that value is invalid, if it is then the passed `auto` values are invalid too.

Reply:
OK, maybe I'm misunderstanding something, so I'll check properly in a local implementation and get back to you.

Reply:
yes @Jaifroid please. And I apologize for taking that much time.

Reply:
No need to apologize!